src/main/java/com/example/translate/controller/GlobalExceptionHandler.java

```diff
@@ -16,4 +17,10 @@ public ResponseEntity<Response> handle(NotFoundException ex){
         Response trs = new Response(404, ex.getMessage());
         return new ResponseEntity<>(trs, HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler(UnexpectedException.class)
+    public ResponseEntity<Response> handle(UnexpectedException ex){
+        Response idk = new Response(500, ex.getMessage());
+        return new ResponseEntity<>(idk,HttpStatus.NOT_FOUND);
```

Comment:
your response status is 500 but HTTP status is 404 or NOT_FOUND, http status should also be 500 use HttpStatus.INTERNAL_SERVER_ERROR